### ISSUE TITLE
Override SYSTEM parameter by commandline

### ISSUE THREAD

**Issue by marco-trovato** [jmorganca/ollama#1415]
According to the [documentation](https://github.com/jmorganca/ollama/blob/main/docs/modelfile.md), the only way to change the SYSTEM is to create a new model with modelfile using an existing LLM model already downloaded as template:

`ollama create choose-a-model-name -f <location of the file e.g. ./Modelfile>'`

But this will copy and duplicate the model file (often bigger than 20 GB)

But using oTerm is possible to change the SYSTEM, please refer to this screenshot for visual reference:
![image](https://github.com/jmorganca/ollama/assets/18162107/42d7b3b6-3e3c-46f1-859d-29755e97ef2a)

REQUEST:
**Please add the --system command line to force system, example usage:**
ollama run codeup:13b-llama2-chat-q4_K_M --verbose **--system** "Roleplay as Matrix movie operator before answering the question." "Write Python code to loop for 1 to 10"

**Comment by marco-trovato** [comment]
> ollama run mistral
> >>> /set system role play as neo from the matrix
> Does that meet your needs?

This is very useful and I didn't know it, thank you. Unfortunately in my specific use case I am trying to use it in **non interactive mode** as command line for a bash script. The solution you proposed works only as interactive and requires the user to actually type the command.

**Comment by iplayfast** [comment]
I agree that having all the possible / command internally in ollama would be useful to have as external parameters when starting the program.

**Comment by cirosantilli** [comment]
Someone semi-asking to set stuff from CLI: https://github.com/ollama/ollama/issues/2505#issuecomment-2435395442
